internal/vendor/wiz.go

```diff
@@ -423,6 +450,20 @@ type KubernetesCluster struct {
 	IsConnectedUsingBroker bool         `json:"isConnectedUsingBroker"`
 }
 
+// ProjectKubernetesClusterLink struct
+type ProjectKubernetesClusterLink struct {
+	ID                string                   `json:"id"`
+	KubernetesCluster ProjectKubernetesCluster `json:"kubernetesCluster"`
```

Comment:
The GraphQL schema contain field kubernetesCluster of type KubernetesCluster.

```suggestion
	KubernetesCluster KubernetesCluster `json:"kubernetesCluster"`
```